sqlite/src/func.c

```diff
@@ -2699,9 +2699,11 @@ void sqlite3RegisterBuiltinFunctions(void){
     FUNCTION2(unlikely,          1, 0, 0, noopFunc,  SQLITE_FUNC_UNLIKELY),
     FUNCTION2(likelihood,        2, 0, 0, noopFunc,  SQLITE_FUNC_UNLIKELY),
     FUNCTION2(likely,            1, 0, 0, noopFunc,  SQLITE_FUNC_UNLIKELY),
+#if !defined(SQLITE_BUILDING_FOR_COMDB2)
 #ifdef SQLITE_DEBUG
     FUNCTION2(affinity,          1, 0, 0, noopFunc,  SQLITE_FUNC_AFFINITY),
 #endif
+#endif /* !defined(SQLITE_BUILDING_FOR_COMDB2) */
```

Comment:
This is good, but IIUC, to make @mistachkin's merge script happy, you'd still need to follow the format I shared in my previous comment. Here's an example:
https://github.com/bloomberg/comdb2/blob/master/sqlite/src/build.c#L212-L217

Reply:
If that's the case we may have a problem because of all the !defined usages, ex. https://github.com/bloomberg/comdb2/blob/master/sqlite/src/analyze.c#L326

Reply:
IC, in that case the script should be able to handle it then. (ping @mistachkin)